app/views/conversations/_show.haml

```diff
@@ -25,5 +25,4 @@
   - for participant in conversation.participants
     = person_image_link(participant, :size => :thumb_small)
 
-.stream
-  = render partial: 'messages', locals: { conversation: conversation }
+= render partial: 'messages', locals: { conversation: conversation }
```

Comment:
- Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
- Space inside { detected.
- Space inside } detected.